examples/3d-tiles/app.js

```diff
@@ -76,6 +76,7 @@ export default class App extends PureComponent {
     this._onViewStateChange = this._onViewStateChange.bind(this);
     this._rotateCamera = this._rotateCamera.bind(this);
     this._getColor = this._getColor.bind(this);
+    this._getControlPanel = this._getControlPanel.bind(this);
```

Comment:
You shouldn't need to bind this method as long as you are calling it via `this.`